src/desktop/assets/auctions.coffee

```diff
@@ -28,7 +28,6 @@ routes =
   /auction-registration/.*
   /auction/.*/bid/.*
   /auction/.*/buyers-premium
-  /feature/.*/bid/.*
```

Comment:
This seems like it _should_ be fine, but this change worries me (is it possible this route is currently active?)

Reply:
I can pull this change off but I was assuming it was for when features supported sales, we'd link to bid pages from within that app — there wasn't a proper auction to fetch so the auction route wouldn't be valid but you'd still want to bid. At least this is what I think I remember.

Reply:
Ahh interesting. I wonder if it's possible to verify (or at least mention to #product-auctions). I suspect you're right but want to be sure.

Reply:
I feel like this route has been marked for removal in the past somewhere - like in a routes.coffee or something - with a note like 'TODO: figure out if we are still using this or remove'. I think i wrote something like this a year or so ago 😬